### PR TITLE
remove deprecated batch

### PR DESCRIPTION
**Pull request by bhakiyakalimuthu**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,22 +1,31 @@
 .PHONY: all build test clean lint cover cover-html up down
-
+include .env
```
Comment (metachris) [inline Makefile:2]: Why this?

File: Makefile [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
+APP_NAME := rpc-endpoint
 GOPATH := $(if $(GOPATH),$(GOPATH),~/go)
-GIT_VER := $(shell git describe --tags --always --dirty="-dev")
+COMMIT := $(shell git rev-parse --short HEAD)
+DATE := $(shell git log -1 --format=%cd --date=format:"%Y%m%d")
+DIFF_HASH := $(shell git diff HEAD -- . | sha1sum | head -c 7)
+EMPTY_DIFF_HASH := da39a3e
+
+GIT_VER := $(COMMIT)-$(DATE)
```
Comment (metachris) [inline Makefile:10]: Why add the date to the version?

**Comment by metachris** [comment]
please split out the changes to the makefile from the other changes (into separate PR)

**Comment by bhakiyakalimuthu** [comment]
@metachris like to keep the docker build and image prep in the make file to make it easier in a single pr. wdyt ?

**Review by metachris — APPROVED** [review]
lgtm

**Pull request merged**